.github/workflows/Build and test.yml

```
@@ -43,71 +43,25 @@
-        uses: jurplel/install-qt-action@v3
-        with:
-          version: 6.6.*
-          host: 'windows'
-          target: 'desktop'
-          arch: 'win64_msvc2019_64'
+      - name: Setup conan
+        run: >
+          pipx install conan
+          pipx run conan profile detect
+          pipx run conan remote add tim-artifactory http://62.250.15.156:8081/artifactory/api/conan/conan-local
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)